README-K8S.md

```diff
@@ -64,6 +64,9 @@ The following are the IPAM attributes:
 - "type" (Required): specifies the plugin type and is also the file name of the plugin executable.
 - "subnet" (Optional): specifies the CIDR to be used for the network. This is a well-known CNI attribute and is used by the driver.
 - "gateway" (Optional): specifies the gateway for the network. This is a well-known CNI attribute and is simply passed through to CNI.
+                        gateway can provide in 0.0.0.x format when subnet is not providing in conf file. but make sure the given gateway value is
```

Comment:
Can you please add this in a Note: like this: 
Note: if subnet is not provided in the conf.then, user needs to follow gateway format as given below:
a) if default prefix-length(24) used then gateway will be in 0.0.0.x format.
b) if prefix-length is provided then user should pass gateway in a away that new created subnet using network-container(default/user configured) should contain the gateway IP.
    Then, give example which shows 0.0.0.x as well as 0.0.y.x. as gateway ip.